.github/workflows/cypress-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Cypress End-to-End test
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+            node-version: '20'
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        with:
+          build: npm run build
+          start: npm start
```

Comment:
Don't know much about the cypress testing part. Better to follow the official docs and figure it out.

Reply:
Yeah I did write it from the official [docs](https://docs.cypress.io/guides/continuous-integration/github-actions). Do have a look at them if you want.

Reply:
Okay :+1: